samcli/local/apigw/local_apigw_service.py

```diff
@@ -92,6 +94,19 @@ def __init__(
         self._dict_of_routes: Dict[str, Route] = {}
         self.stderr = stderr
 
+        self._click_session_id = None
```

Comment:
I think this session_id should be similar to the default session_id we already use in the Event Tracker. can we remove this part here, and just depend on the default session_id used in the event tracker

Reply:
Addressed in above comment thread ^^